smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -418,7 +420,12 @@ public void periodic() {
     else { // SwerveDriveMode.NORMAL_DRIVING
       // take the current 'requested' chassis speeds and ask the ask the swerve modules to attempt this
       // first we build a theoretical set of individual module states that the chassisSpeeds would corespond to
-      states = swerveKinematics.toSwerveModuleStates(chassisSpeeds);
+      if (swerveDriveRotationMode == SwerveDriveRotationMode.FinePlacement) {
+        states = swerveKinematics.toSwerveModuleStates(chassisSpeeds, Constants.FinePlacementRotationalCenter);
```

Comment:
I'm thinking I would have used else if and placed this logic in 3 options ... x, fine, normal (that would be more readable to me)

Reply:
Also ... I need to understand what it is that the FinePlacementRotationalCenter does when passed to toSwerveModuleStates ... help me understand?  I'm guessing it changes centroid of robot rotation or something?

Reply:
Yes.  That's what it does.  Will rename the enum to make that more clear (see comment above) and leave a comment here in the code.

Reply:
changed enum name. let me know if the intent is now clear.